src/react/components/common/confirm/confirm.tsx

```diff
@@ -16,6 +16,7 @@ export interface IConfirmProps extends IMessageBoxProps {
     cancelButtonText?: string;
     confirmButtonColor?: string;
     cancelButtonColor?: string;
+    className?: string;
```

Comment:
className should be intrinsic on all react components... You may just need to ensure the component interface is extending the `React.Props` interface.

Reply:
@wbreza what if it already extends the `React.Component` interface? Can't it only extend one of these?